source/upgrade/important-upgrade-notes.rst

```diff
@@ -2,14 +2,45 @@ Important Upgrade Notes
+|                                                    |                                                                                                                                                                  |
+|                                                    | 2. Medium effort, medium downtime - This process will require SQL queries to be executed manually on the server. To avoid causing a table lock, a customer can   |
+|                                                    | choose to use the pt-online-schema-change tool for MySQL. For Postgres, the table locking is very minimal. The advantage is that since there are a lot of        |
+|                                                    | queries, the customer can take their own time to run individual queries during off-hours. All queries except #11 are safe to be executed this way. Then the      |
+|                                                    | usual method of (1) can be followed.                                                                                                                             |
+|                                                    |                                                                                                                                                                  |
+|                                                    | 3. High effort, low downtime - This process requires everything of (2), plus it tries to minimize the impact of query #11. We can do this by following step 2,   |
+|                                                    | and then starting v6 along with a running v5 server, and then monitor the application logs. As soon as the v6 application starts up, we need to bring down a v5  |
+|                                                    | node. This is a delicate dance, but minimizes the downtime to only a few seconds.                                                                                |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Please see `the changelog <https://docs.mattermost.com/install/self-managed-changelog.html>`_ for a list deprecations in this release.                           |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Focalboard v0.9.1 (released with Mattermost v6.0) requires Mattermost v5.38+.                                                                                    |
```

Comment:
This has since been updated to:
"Focalboard plugin has been renamed to Mattermost Boards, and v0.9.1 (released with Mattermost v6.0) requires Mattermost v6.0+."